public/app/controllers/mainCtrl.js

```diff
@@ -4,23 +4,20 @@ angular.module('kibibitCodeEditor')
   '$scope',
   '$http',
   'ngDialog',
-  'FileService',
-  'FolderService',
   'Fullscreen',
   'SettingsService',
+  'EventManagerService',
   function(
     $scope,
     $http,
     ngDialog,
-    FileService,
-    FolderService,
     Fullscreen,
-    SettingsService) {
+    SettingsService,
+    EventManagerService) {
 
     var vm = this;
 
-    // Init
-    vm.code = '';
+    vm.selectedFilePath = '';
```

Comment:
I don't like this variable name, just because it will soon be an array of file paths (all open tabs).
because of that, it makes sense to me that this should be something like `openFile`.

later, this will be
```javascript
var openFiles = ['file_path1', 'file_path2'];
```